symplyphysics/laws/dynamics/work_done_by_general_force_in_one_dimension.py

```diff
@@ -0,0 +1,60 @@
+from sympy import Eq, Integral, symbols, solve
+from symplyphysics import (
+    units,
+    Quantity,
+    Symbol,
+    Function,
+    print_expression,
+    validate_input,
+    validate_output,
+)
+
+# Description
+## Assuming one-dimensional environment, when the force F on a particle-like object depends
+## on the position of the object, the work done by F on the object while the object moves
+## from one position to another is to be found by integrating the force along the path of the
+## object.
+
+# Law: W = Integral(F(x), x)
+## W - work done by force F
+## F = F(x) - force in question
+## x - position of the object on the axis
+
+work = Symbol("work", units.energy)
+force = Function("force", units.force)
+position = Symbol("position", units.length)
+
+law = Eq(work, Integral(force(position), position))
```

Comment:
I think it is better to add integration limits to the law, eg `Integral(force(position), (position, position_start, position_end))`

Reply:
I was unsure at first, but it probably is better to do so indeed.